examples/vars.yml

```diff
@@ -0,0 +1,25 @@
+postgres_password: "{{ lookup('password', 'inventory/host_vars/{{domain}}/passwords/postgres.psk chars=ascii_letters,digits') }}"
+
+# You can set any pict-rs environmental variables here. They will populate the templates/docker-compose.yml file.
+# https://git.asonix.dog/asonix/pict-rs
+pictrs_env_vars:
+  - PICTRS__SERVER__API_KEY: "{{ postgres_password }}"
+  - PICTRS__MEDIA__VIDEO_CODEC: vp9
+  - PICTRS__MEDIA__GIF__MAX_WIDTH: 256
+  - PICTRS__MEDIA__GIF__MAX_HEIGHT: 256
+  - PICTRS__MEDIA__GIF__MAX_AREA: 65536
+  - PICTRS__MEDIA__GIF__MAX_FRAME_COUNT: 400
+  - PICTRS_OPENTELEMETRY_URL: http://otel:4137
+  - RUST_LOG: debug
+  - RUST_BACKTRACE: full
```

Comment:
Should we really have the open telemetry, debug mode, and full backtraces enabled for production? 

I think we should make these 3 commented as well

Reply:
Can you submit a new issue for this? I can see an argument for both:

* Keeping it enabled is excessive and noisy/unnecessary

* Enabling it by default gives useful debug for users/developers who may not be savvy enough to know how to adjust this environment to get useful debug output without some finagling